src/ucp/proto/proto_init.c

```diff
@@ -170,7 +170,8 @@ ucp_proto_init_add_tl_perf(const ucp_proto_common_init_params_t *params,
 
     /* Send time is representing request completion, which in case of zcopy
        waits for ACK from remote side. */
-    if (params->flags & UCP_PROTO_COMMON_INIT_FLAG_SEND_ZCOPY) {
+    if ((op_attr_mask & UCP_OP_ATTR_FLAG_FAST_CMPL) &&
```

Comment:
why is this overhead relevant for fast completion case only? Can it be that some other estimation is wrong for non-fast completion cases?

Reply:
@brminich, I initially tried to revert to a stage where latency overhead is added to FAST_CMPL as well (https://github.com/openucx/ucx/pull/10213/files),
but now that I think about it we might want to separate the two scenarios (send zcopy ack overhead vs. fast completion latency taken into account on the remote side), so it would look something like this:

```suggestion
    if (params->flags & UCP_PROTO_COMMON_INIT_FLAG_SEND_ZCOPY) {
        perf_factors[UCP_PROTO_PERF_FACTOR_LATENCY].c += tl_perf->latency;
    }

    if (op_attr_mask & UCP_OP_ATTR_FLAG_FAST_CMPL) {
        perf_factors[UCP_PROTO_PERF_FACTOR_LATENCY].c += tl_perf->latency;
    }
```